README.md

```diff
@@ -49,6 +49,7 @@ The table below lists recommender algorithms available in the repository at the
 | [Smart Adaptive Recommendations (SAR)<sup>*</sup>](notebooks/00_quick_start/sar_movielens.ipynb) | Python CPU | Collaborative Filtering | Similarity-based algorithm for implicit feedback dataset |
 | [Surprise/Singular Value Decomposition (SVD)](notebooks/02_model/surprise_svd_deep_dive.ipynb) | Python CPU | Collaborative Filtering | Matrix factorization algorithm for predicting explicit rating feedback in datasets that are not very large | 
 | [Vowpal Wabbit Family (VW)<sup>*</sup>](notebooks/02_model/vowpal_wabbit_deep_dive.ipynb) | Python CPU (train online) | Collaborative, Content-Based Filtering | Fast online learning algorithms, great for scenarios where user features / context are constantly changing |
+| [LightGBM/Gradient Boosting Tree<sup>*</sup>](notebooks/00_quick_start/lightgbm_tinycriteo.ipynb) | Python CPU | Content-Based Filtering | Gradient Boosting Tree algorithm for fast training and low memory usage in content-based problems |
```

Comment:
Can we order the table alphabetically? I created an issue #663

Reply:
We are now sorting as CPU->GPU->Spark. Do you mean to order alphabetically the algos within each env?

Reply:
Hmmmm... Let’s say one comes to find an algo for his problem:
1. Maybe already knows what he wants to use (algo name) but just looking for an implementation example of that algo. Then alphabetical sort will work for him.
2. Have no idea which algo to use, but knows about his problem which doesn’t have much historical data but bunch item features. Then algo types will be important to him and will skip CF-algos.

But I don’t see much benefit of sorting by CPU/GPU/spark, especially you can use CPU for GPU algos and vice versa anyway. Also you can use Databricks (spark) for CPU and GPU algos.

Reply:
Maybe it would be good to sort by `type` which is informative to users regarding different use cases.

Reply:
Github please give us interactive table...

Reply:
Github please give us interactive table...

Reply:
Sorting first by type then by env seems like a good approach.